backend/src/main/java/capstone/myfinancemanager/manager/service/TransactionService.java

```diff
@@ -61,4 +65,19 @@ public boolean deleteTransaction(String id) {
         }
         return false;
     }
+
+
+    public Transaction updateTransaction(String id, TransactionCreationDto transactionUpdate) {
+
+        Transaction transactionToUpdate = transactionRepo.findById(id).orElseThrow(() -> new ResponseStatusException(HttpStatus.FORBIDDEN));
```

Comment:
Good use of orElseThrow nut 403 FORBIDDEN means the authorized user is missing priviledge to perform the requested operation. 404 NOT FOUND would be a better fit as we can not update something that is not present.